includes/class-wc-gateway-ppec-admin-handler.php

```diff
@@ -168,6 +168,11 @@ public function capture_payment( $order_id ) {
 					$order->add_order_note( __( 'Unable to capture charge!', 'woocommerce-gateway-paypal-express-checkout' ) . ' ' . $result->get_error_message() );
 				} else {
 					update_post_meta( $order_id, '_paypal_status', ! empty( $trans_details['PAYMENTSTATUS'] ) ? $trans_details['PAYMENTSTATUS'] : 'completed' );
+
+					if ( ! empty( $trans_details['TRANSACTIONID'] ) ) {
+						update_post_meta( $order_id, '_paypal_status', $trans_details['TRANSACTIONID'] );
```

Comment:
How come transactionid is going into "_paypal_status" meta? Shouldn't it be "_transaction_id" ?